pkg/apis/config/feature_flags.go

```diff
@@ -94,6 +94,8 @@ const (
 
 // FeatureFlags holds the features configurations
 // +k8s:deepcopy-gen=true
+//
+//nolint:musttag
```

Comment:
`musttag` seems to have a lot of things disable it. Is it worth just not having? Or is there some way to make it only apply to certain directories (`pkg/apis`-but-not-`_test.go`, probably?) instead of opting out a lot?

I don't think most types care about having `json` tags except stuff in `pkg/apis`.

Reply:
Unfortunately, there's no "opt-in" mechanism, only "opt-out." 🙈

The most pedantic approach would be to fix the identified `musttag` violations rather than opting them out; every instance of `nolint` really can be considered a `TODO`.